src/bosh-director/spec/unit/bosh/director/jobs/fetch_logs_spec.rb

```
@@ -38,11 +38,11 @@ module Bosh::Director
           end
 
           before { allow(AgentClient).to receive(:with_agent_id).with(instance.agent_id, instance.name).and_return(agent) }
-          let(:agent) { instance_double('Bosh::Director::AgentClient', fetch_logs: {'blobstore_id' => 'new-fake-blobstore-id'}) }
+          let(:agent) { instance_double('Bosh::Director::AgentClient', fetch_logs: {'blobstore_id' => 'aaaaaaaa-aaaa-4aaa-8aaa-aaaaaaaa0001'}) }
 
           it 'cleans old log bundles' do
-            old_log_bundle = FactoryBot.create(:models_log_bundle, timestamp: Time.now - 12*24*60*60, blobstore_id: 'previous-fake-blobstore-id') # 12 days
-            expect(blobstore).to receive(:delete).with('previous-fake-blobstore-id')
+            old_log_bundle = FactoryBot.create(:models_log_bundle, timestamp: Time.now - 12*24*60*60, blobstore_id: 'bbbbbbbb-bbbb-4bbb-8bbb-bbbbbbbbb001') # 12 days
```

RuboCop (1.86.1)

[warning] Wrap expressions with varying precedence with parentheses to avoid ambiguity.

(Lint/AmbiguousOperatorPrecedence)